pages/_middleware.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { NextRequest, NextResponse } from "next/server";
+
+// FIXME - this would be nicer if it was dynamic using `fs.readDirSync('.')
```

Comment:
I guess this [is a nope](https://nextjs.org/docs/api-reference/edge-runtime#unsupported-apis):

> Native Node.js APIs are not supported. For example, you can't read or write to the filesystem